firebase-firestore/src/main/java/com/google/firebase/firestore/remote/WatchStream.java

```diff
@@ -100,7 +100,12 @@ public void unwatchTarget(int targetId) {
   }
 
   @Override
-  public void onNext(com.google.firestore.v1.ListenResponse listenResponse) {
+  public void onFirst(ListenResponse listenResponse) {
+    onNext(listenResponse);
```

Comment:
Is there a reason why watch stream does not handle initialization or is it in one of the next prs?